design/FY2024/NFP-Space Sizing and HVAC-Part4.md

```diff
@@ -0,0 +1,191 @@
+Extend Spaces to Sizing and HVAC - Part 4
+================
+
+**Michael J. Witte, GARD Analytics, Inc.**
+
+ - Original June 17, 2024
+
+## Table of Contents ##
+
+[E-mail and Conference Call Conclusions](#e-mail-and-conference-call-conclusions)
+
+[Background and Overiew](#background-and-overview)
+
+[Approach](#approach)
+
+[Testing/Validation/Data Sources](#testingvalidationdata-sources)
+
+[Input Description](#input-description)
+
+[Outputs Description](#outputs-description)
+
+[Engineering Reference](#engineering-reference)
+
+[Example File and Transition Changes](#example-file-and-transition-changes)
+
+[Design](#design)
+
+## E-mail and Conference Call Conclusions ##
+
+
+## Background and Overview ##
+
+Space was added as a new concept in v9.6. Each EnergyPlus Zone contains one or more Spaces which are used for:
+
+  * assigning and allocating internal gains
+  * specifying enclosure boundaries,
+  * reporting inputs grouped by space types, and 
+  * reporting select output grouped by space types.
+  
+For the zone heat balance each thermal Zone is composed of one or more Spaces controlled by a single thermostat or HVAC system control point such as a VAV terminal unit.
+
+In version 23.1 options were added to perform the air heat balance for each Space or each Zone, and space-level heat balance output variables were added.
+
+In version 23.2, the following capabilities were added:
+  
+  * When ZoneAirHeatBalanceAlgorithm "Do Space Heat Balance for Sizing" = Yes, zone sizing is also done for all spaces. The HVAC Sizing Summary table report will include subtables for Space Sensible Cooling and Heating as well as for Zone Sensible Cooling and Heating. Space Sizing will also be reported to the eio output. The space sizing results are reported, but not used.
+
+  * Three new objects, SpaceHVAC:EquipmentConnections, SpaceHVAC:ZoneEquipmentSplitter, and SpaceHVAC:ZoneEquipmentMixer allow one or more zones to be simulated at the space level for HVAC, leading to independent air temperature and humidity in each space.
+  
+  * For zone equipment serving multiple spaces, three thermostat control options (SingleSpace, Ideal, and Maximum).
+
+This NFP proposes additional optional capabilities:
+
+  * Use the Space-level (room-by-room) sizing results to size Zone-level equipment to either the coincident or non-coincident peak across the Spaces (rooms).
+  
+  * Refine existing Space-level HVAC simulation.
+  
+  * Extend space HVAC to support more special objects (e.g. ZoneThermalChimney, if budget allows).
```

Comment:
A reasonable improvement for spaces. Shouldn't the new field Type of Space Sum to Use include an option to revert back to zone sizing (e.g., None)? and be the default? I guess I'm not clear on what trigger allows spaces, instead of zones, to size zone equipment, and what zone equipment is targeted here.

Reply:
My thought was that the coincident option would use the current zone (lumped spaces) result. 

I should highlight here that the same thermostat setting is applied to the zone and its spaces. So, it's not possible in this plan to have a space within a zone set to a different temperature, for example.

Reply:
OK, and hopefully sum of space coincident sizing adds up to what zone sizing shows within some small tolerance. I don't see why it wouldn't but you never know (space wall loads?, diffs in window solar gains? etc., but I guess there should still be a similar space sum).

Reply:
Simple question: If you can sum space loads to currently give a zone load with existing code, with known space fractions of all loads, and that answer is accurate, what benefit would it be to perform sizing on spaces? Wouldn't the space sum just be a sum of the original fractions used to create a zone load? I almost get the non-coincident part but the coincident part seems to mean no space sizing is necessary (i.e., skip it). And it even seems the non-coincident sizing is just a sum of a different time in a series of data, which then would also mean no space sizing is necessary. If coincident gives the same answer as zone sizing then non-coincident just sums data from different times from that same time series data of zone sizing that is fractionally allocated to spaces. So my question is "what is meant by space sizing"? What heat balance will be performed? is it a sum of some fractions of space allocations, or is it an actual heat balance on a space? Which would then be just a subset of zone sizing with known fractions. I'm trying to think of a benefit here but not coming up with a good answer.

    Lights,
      A1 , \field Name
      A2 , \field Zone or ZoneList or Space or SpaceList Name

Reply:
Space sizing is an actual heat balance on each space. Currently zone sizing is an actual heat balance on each zone (as a whole) although some of the components for the zone heat balance are sums across the spaces (even when space heat balance is off). e.g. internal gains.

For the coincident zone sizing, I was planning to just use the current lumped zone results rather than combine the space-level results into that. But I should at least check that calculating it from space results would give the same answer. It *should* be the same since the spaces are all using the same thermostat setpoint. Hmmm, but what if one space is floating while others are calling for cooling, or worse yet, what if one space needs heat and another needs cooling? But conceptually, the zone sizing should be the lumped result when choosing coincident space sums.

Reply:
I was running 5ZoneAirCooledWithSpacesHVAC last night in 24.1 and looking at results to try to get a better feel for space sizing. I see 5ZoneAirCooledWithSpacesHVACZsz.csv (and Ssz) but no 5ZoneAirCooledWithSpacesHVACSpsz for spaces so I could not see how the space temps reported during the peak time. The space temps during sizing should be at or very near the Tstat temp but if not doing latent sizing w may be very different. Adding latent sizing proved changing zone w didn't change sizing results much. I'm not sure but I wonder if the zone sizing results and the different space fractions of all loads could hold all the info needed for summing/reporting space loads. If space temps are actually different, as I expect they would be, then the heat balance is needed during the simulation but I wonder about sizing. Probably won't save much execution speed so not worth the effort. I just wanted to think through that thought a little while talking about spaces.